Build with `FMILIB_BUILD_SHARED_LIB=OFF` can't find `#include` (regression in 3.0.4)

Trying to build release 3.0.4 on Linux with **`-DFMILIB_BUILD_SHARED_LIB=OFF`** fails to build the tests:
````sh
$ git clone -q --depth 1 --branch 3.0.4 https://github.com/modelon-community/fmi-library.git ../fmi-library
$ cmake ../fmi-library/ -DFMILIB_BUILD_SHARED_LIB=OFF
[...]
$ cmake --build .
[...]
[ 53%] Building C object CMakeFiles/fmi_import_test.dir/Test/fmi_import_test.c.o
In file included from ~/fmi-library/src/Import/include/FMI/fmi_import_context.h:30,
                 from ~/build/fmilib_include/fmilib.h:27,
                 from ~/fmi-library/Test/fmi_import_test.c:21:
~/fmi-library/src/Util/include/FMI1/fmi1_types.h:44:10: fatal error: FMI1/fmiPlatformTypes.h: No such file or directory
   44 | #include <FMI1/fmiPlatformTypes.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
````

The reason is apparently typos in the static library target include directories. This fixes the build:
````diff
index e4c92fb..edaf36d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -338,8 +338,8 @@ if(FMILIB_BUILD_STATIC_LIB)
     # the same include dirs as will exist for installation.
     target_include_directories(fmilib
         PUBLIC
-            ${JMUTIL_PUBLIC_INCLUDE_DIR}
-            ${FMIIMPORT_PUBLIC_INCLUDE_DIR}
+            ${JMUTILS_PUBLIC_INCLUDE_DIRS}
+            ${FMIIMPORT_PUBLIC_INCLUDE_DIRS}
             ${FMILIB_INCLUDE_DIR}
             ${FMILIB_CONFIG_INCLUDE_DIR}
     )
````

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -338,8 +338,8 @@ if(FMILIB_BUILD_STATIC_LIB)
     # the same include dirs as will exist for installation.
     target_include_directories(fmilib
         PUBLIC
-            ${JMUTIL_PUBLIC_INCLUDE_DIR}
-            ${FMIIMPORT_PUBLIC_INCLUDE_DIR}
+            ${JMUTILS_PUBLIC_INCLUDE_DIRS}
+            ${FMIIMPORT_PUBLIC_INCLUDE_DIRS}
             ${FMILIB_INCLUDE_DIR}
             ${FMILIB_CONFIG_INCLUDE_DIR}
     )

